tasks/main.yml

```diff
@@ -75,14 +75,16 @@
   file:
     path: "{{ ler53_cert_dir }}"
     state: directory
-    owner: root
+    owner: "{{ ler53_cert_files_owner }}"
```

Comment:
Could you please document these new variables in the read me?

Reply:
Variables ler53_cert_files_owner and ler53_cert_files_group already existed and are documented in README. For some reason, they were not used for directories. This commit fixes it.